tools/roslaunch/src/roslaunch/launch.py

```diff
@@ -540,7 +540,7 @@ def launch_node(self, node, core=False):
         except roslaunch.node_args.NodeParamsException as e:
             self.logger.error(e)
             if node.package == 'rosout' and node.type == 'rosout':
-                printerrlog("\n\n\nERROR: rosout is not built. Please run 'rosmake rosout'\n\n\n")
+                printerrlog("\n\n\nERROR: rosout is not built. Please run 'catkin_make rosout'\n\n\n")
```

Comment:
In that case, let's just drop the special case; the generic one is just as applicable to rosout being missing.